recipes/msys2/20190524/LICENSE.md

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2013, Алексей
```

Comment:
All Conan recipes within the Conan Center Index need to be licensed under the MIT license,
also we aren't adding individual licenses anymore

Reply:
Thanks for the quick response. The recipe was copied from the one previously in conan-center index and I added it merely by creating a new version. I'd see two options forward:
1. Refactor the existing recipe to serve both versions (I'm not sure if this improves the licensing situation)
2. Rewrite the recipe under a new License

Can you recommend how to proceed, as I'm not yet familiar with how this situation was handled in previous cases in conan-center-index?

Reply:
It seems like the download package of msys2  doesn't contain a license file itself, so it got manually added here.

My mistake, sorry!

Reply:
it would be much better to use the same recipe for both versions